### ISSUE TITLE
Option for live reload of .sql files

### ISSUE THREAD

**Issue by f8dca** [lovasoa/SQLpage#47]
It appears that the .sql files are read when the executable starts. It would be very convenient to have a live reload option that would allow changing the .sql files without an application restart.

**Comment by lovasoa** [comment]
No, sqlpage are not loaded when the executable starts, but they stay in cache for a few milliseconds before being reloaded from disk, for better performance.

If you don't see the stale files for more than a few milliseconds, it's a bug.

How long are you observing stale contents being served ?

**Comment by Bungle50** [comment]
I am noticing this behavior when files are remotely edited, e.g. over a network share or OneDrive folder. The stale contents are served until an application restart. Editing the .sql on the local machine directly does make the changes show instantly.

**Comment by lovasoa** [comment]
sqlpage checks whether files have been modified before reloading them; if the filesystem does not accurately reflect file modification times, then it will serve stale files.

If you want to collaborate with others on a sqlpage app, I can recommend using git instead of a onedrive folder.
git is made for sharing code: you'll also never lose code again, or override your collaborators' work.

https://github.com/sqlpage/SQLPage/blob/main/src/file_cache.rs#L138
https://github.com/sqlpage/SQLPage/blob/main/src/filesystem.rs#L203
